policy/release_manager.md

```diff
@@ -0,0 +1,49 @@
+# Release manager role
+
+## Appointment
+
+A release manager (RM) is appointed for a specific upcoming release.
+
+A RM may and should step down at any time, when they cannot fulfill the associated obligations. In this case, a new RM is appointed.
+
+The appointment of an RM ends with the declaration of the respective release's end-of-life by the RM.
+
+## Responsibilities
```

Comment:
is making the actual release, including Changelog generation, also part of the responsibilities?

Reply:
I'd say: There cannot be a release without a changelog. I think it would not be useful to state that a RM needs to personally craft one, but they obviously should make sure that one is made by some processes.

If that changelog is bad, it can be fixed, and complained about (like any other issue), but this is entirely orthogonal to the RM role definition.